library/commandline/src/modules/CommandLine.rb

```diff
@@ -1556,6 +1521,7 @@ def Run(commandline)
           # there is a handler, execute the action
           if !exec.nil?
             res = exec.call(options)
+            read_only = false unless commandline["actions"][command]["readonly"]
```

Comment:
This means that a missing value (-> `nil`) is the same as `false`. Sounds right but why default to `true` above?

Reply:
as I said, if there is no operation, then it is readonly, but I am not sure if it can happen. And also it is easier logic for me in interactive mode, otherwise I need tri-state - not set, read_only commands only and at least one write command.